src/plugins/robots/drone/hardware/drone_flight_system_default_sensor.cpp

```diff
@@ -86,10 +86,24 @@ namespace argos {
          /* set the initial position if not already set */
          if(!m_pcPixhawk->GetInitialPosition()) {
             m_pcPixhawk->GetInitialPosition().emplace(m_cPosition);
-         }
+         } 
```

Comment:
I'm not sure which editor you use, but if you use something like Visual Studio Code, it will highlight whenever you accidentally introduce noise into the code.